requirements/pip.txt

```diff
@@ -100,7 +100,7 @@ django-messages-extends==0.6.0
 djangorestframework-jsonp==1.0.2
 # django-taggit 0.23.0 drops support for django < 1.11
 django-taggit==0.22.0
-dj-pagination==2.3.2
+dj-pagination==2.4.0
```

Comment:
dj-pagination removes support for django <1.11 https://github.com/pydanny/dj-pagination/commit/92306587a601e6b69e77b809cc49bca66fac8c21

I didn't see anything relevant, but still just to be sure we should wait for the django upgrade